pkg/proxy/service.go

```diff
@@ -213,10 +213,16 @@ func (sct *ServiceChangeTracker) newBaseServiceInfo(port *v1.ServicePort, servic
 		if ing.IP == "" {
 			continue
 		}
-		if ipFamily := proxyutil.GetIPFamilyFromIP(ing.IP); ipFamily == sct.ipFamily && proxyutil.IsVIPMode(ing) {
+		if ipFamily := proxyutil.GetIPFamilyFromIP(ing.IP); ipFamily != sct.ipFamily {
```

Comment:
I'm not sure here - if the ingress IP is "proxy mode" then it could very well be IPv6 while the backends are IPv4.  I think it's only an error if it is VIP.  Or else I misunderstand...

Reply:
Yeah, I remember both Google and AWS has 6to4 load balancers, I'll change it

Reply:
done